### ISSUE TITLE
snap_query_version does not seem to work right

### ISSUE THREAD

**Issue by tstack** [Lin-Buo-Ren/selective-checkout#3]
The `snap_query_version` function does not seem to be working and just returns an empty string for any queries.  I think the `awk` script is incompatible with the `snap info` output.  The output that I see is:

```
channels:
  latest/stable:    0.8.5                 2020-04-20 (1163) 12MB -
  latest/candidate: 0.8.5                 2020-09-24 (1525) 13MB -
  latest/beta:      0.8.5                 2020-09-24 (1525) 13MB -
  latest/edge:      0.9.0-rc3-14-gc3646be 2020-09-24 (1542) 17MB -
```

The awk script doesn't seem to account for the `latest/` part:

```
awk "\$1 == \"${release_channel}:\" { print \$2 }"
```

**Comment by Lin-Buo-Ren** [comment]
Thanks for the report!  I've notice the problem recently as well.

This should be fixed by switching to call the Snap Store's API, which should provide consistent result in the near future.

**Comment by Lin-Buo-Ren** [comment]
Fixed in commit 20d748f041d88c237bba95205d3f2ce94aaaf00e, please refer [Selective-checkout: Check out the tagged release revision if it isn't promoted to the stable channel - doc - snapcraft.io](https://forum.snapcraft.io/t/selective-checkout-check-out-the-tagged-release-revision-if-it-isnt-promoted-to-the-stable-channel/10617) for the new part definition required for the change.
